llvm/docs/ReleaseNotes.md

```diff
@@ -165,7 +165,8 @@ Changes to the LLVM tools
 
 * llvm-objcopy now supports the `--update-section` flag for intermediate Mach-O object files.
 * llvm-strip now supports continuing to process files on encountering an error.
-* In llvm-objcopy's ELF port, `--discard-locals` and `--discard-all` now allow and preserve symbols referenced by relocations.
+* In llvm-objcopy/llvm-strip's ELF port, `--discard-locals` and `--discard-all` now allow and preserve symbols referenced by relocations.
+  (`#47468 <https://github.com/llvm/llvm-project/issues/47468>`_).
```

Comment:
I'm not entirely sure of the syntax but do you mean:

`([#47468](https://github.com/llvm/llvm-project/issues/47468))`

Reply:
Thanks for spotting this! I thought I was writing reST..